### PR TITLE
[MERGE] dialog into development

### PR DESCRIPTION
**Pull request by LotteBase**
Dialog component, to test:
[ ] - dialog works with given string input
[ ] - dialog works with given audio
[ ] - dialog component looks usably clear for creators
[ ] - dialog delay works when auto skip is enabled
[ ] - dialog clip end also skips when skip enabled

### CODE REVIEW

**Review by Vanclyf — COMMENTED** [review]
Maybe it's nice to have the option to execute a unity event on a specific dialog trigger

File: Ravel-Creators/Assets/RavelCreator/Scripts/Editor/CreatorTools/Assetbundles/BundleBuilder.cs [hunk 1]
```diff
@@ -80,15 +89,25 @@ public static class BundleBuilder
...
+		}
+		
 		//Last chance for user to cancel. This dialog shows what scene is assigned into what environment.
-		if (!EditorUtility.DisplayDialog("Build confirmation",
-			    $"Building scene {s.name} and uploading it to environment " +
-			    $"{config.environmentSO.environment.name}, this will override the previous assetbundle. Are you sure?",
-			    "Yes", "Cancel build")) {
+		if (!EditorUtility.DisplayDialog("Build confirmation", dialogMsg, "Yes", "Cancel build")) {
 			return;
 		}
```
Comment (Vanclyf) [inline Ravel-Creators/Assets/RavelCreator/Scripts/Editor/CreatorTools/Assetbundles/BundleBuilder.cs]: var assetBundleManifest = BuildPipeline.BuildAssetBundles(path, BuildAssetBundleOptions.None,
			BuildTarget.WebGL); is unused, please remove the variable

**Review by Vanclyf — COMMENTED** [review]
Trying to publish an environment results in a memory leak and the webgl player bre
image
aking

**Review by Vanclyf — COMMENTED** [review]
Hij zette de router uit en aan, niet aan en uit. dan werkt de router niet. adoi (je mag dit negeren)

**Review by Vanclyf — COMMENTED** [review]
image

In a preview build, I get a memory out of bounds error

File: Ravel-Creators/Assets/RavelCreator/Scripts/Editor/CreatorTools/Assetbundles/BundleBuilder.cs (hunk 1 above)
Comment (LotteBase) [reply Ravel-Creators/Assets/RavelCreator/Scripts/Editor/CreatorTools/Assetbundles/BundleBuilder.cs]: in andere branch gedaan haha

**Review by Vanclyf — APPROVED** [review]

**Pull request merged**